sos/report/plugins/process.py

```diff
@@ -21,7 +21,9 @@ class Process(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin, CosPlugin):
         ("lsof", "gathers information on all open files", "slow", True),
         ("lsof-threads", "gathers threads' open file info if supported",
          "slow", False),
-        ("smaps", "gathers all /proc/*/smaps files", "", False)
+        ("smaps", "gathers all /proc/*/smaps files", "", False),
+        ("samples", "specify the number of samples that iotop will capture, "
+            "with an interval of 0.5 seconds between samples", "", "")
```

Comment:
Can't you provide the `20` as the default value directly here? (the latest argument/item in the plugopt list).

Reply:
You are right, I oversaw that possibility. Let me resend with this change